docs/api.rst

```diff
@@ -444,12 +444,12 @@ Querying general network events
 
 
 You can query for non-channel specific events by making a ``GET`` request to the
-endpoint of the token registry contract. ``GET /api/<version>/events/network/<token_registry_address>``
+following endpoint. ``GET /api/<version>/events/network``
```

Comment:
As we discussed in Spain, we will force a default registry for now which will probably be part of some raiden config value. Yes I will improve the docstring though as it may be misleading.